source/includes/_090-properties.md.erb

```diff
@@ -0,0 +1,8 @@
+# Properties API
+
+
+        You can get the list of properties or a property's value for a given job by using the properties API.
```

Comment:
Kill the leading whitespace, different markdown implementations will interpret it differently. Also, I'd probably keep it short to "The properties API allows managing of job properties"